python/tk_alias/menu_generation.py

```diff
@@ -188,22 +188,40 @@ def _jump_to_fs(self):
             if exit_code != 0:
                 self._engine.logger.error("Failed to launch '%s'!", cmd)
 
-    def _version_check(self, version):
+    def _version_check(self, version1, version2):
         """
         Check if Alias version is greater than 2022.2 for ShotGrid menu name
```

Comment:
We should update the function doc comment to match what the function does, something like:
```
Compare version strings and return 1 if version1 greater than version2, 0 if they are equal and -1 if version1 less than version2.
```